tendrl/node_agent/ansible_runner/ansible_module_runner.py

```diff
@@ -0,0 +1,78 @@
+import ansible.executor.module_common as module_common
```

Comment:
Remove this file from the PR, since we just want to merge the framework bits and not the ansible task related bits